ncm-metaconfig/src/main/metaconfig/keepalived/global.tt

```diff
@@ -0,0 +1,3 @@
+[% FOREACH item IN global_defs.pairs -%]
+[%     item.key %] [% item.value -%]
```

Comment:
the ending `-%]` will put multiple global_defs on the same line, make sure your test example has more than on entry